oneview_redfish_toolkit/app.py

```diff
@@ -66,6 +69,68 @@
 app.register_blueprint(thermal)
 app.register_blueprint(storage_collection)
 
+
+@app.errorhandler(status.HTTP_400_BAD_REQUEST)
+def bad_request(error):
+    """Creates a Bad Request Error response"""
+    redfish_error = RedfishError(
+        "PropertyValueNotInList", error.description)
+
+    redfish_error.add_extended_info(
+        message_id="PropertyValueNotInList",
+        message_args=["VALUE", "PROPERTY"],
+        related_properties=["PROPERTY"])
+
+    error_str = redfish_error.serialize()
+    return Response(
+        response=error_str,
+        status=status.HTTP_400_BAD_REQUEST,
+        mimetype='application/json')
+
+
+@app.errorhandler(status.HTTP_404_NOT_FOUND)
+def not_found(error):
+    """Creates a Not Found Error response"""
+    redfish_error = RedfishError(
+        "GeneralError", error.description)
+    error_str = redfish_error.serialize()
+    return Response(
+        response=error_str,
+        status=status.HTTP_404_NOT_FOUND,
+        mimetype='application/json')
+
+
+@app.errorhandler(status.HTTP_500_INTERNAL_SERVER_ERROR)
+def internal_server_error(error):
+    """General InternalServerError handler for the app"""
```

Comment:
Nit but this comment is not worded in the same way as the other errorhandlers

Reply:
fixed